charts/opensearch-dashboards/CHANGELOG.md

```diff
@@ -358,7 +368,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 ### Security
 
-[Unreleased]: https://github.com/opensearch-project/helm-charts/compare/opensearch-dashboards-2.20.0...HEAD
+[Unreleased]: https://github.com/opensearch-project/helm-charts/compare/opensearch-dashboards-2.20.1...HEAD
+[2.20.1]: https://github.com/opensearch-project/helm-charts/compare/opensearch-dashboards-2.20.0...opensearch-dashboards-2.20.1
```

Comment:
This should be 2.21.0 as it is a feature not a patch, so need to bump the minor version imho.

Thanks.